corehq/apps/registry/fixtures.py

```diff
@@ -21,7 +21,7 @@ def __call__(self, restore_state):
             module.search_config.data_registry
             for app in apps
             for module in app.get_modules()
-            if module.search_config.data_registry
+            if hasattr(module, 'search_config') and module.search_config.data_registry
```

Comment:
There's a `module_offers_search` function for this, which also checks if the search config is active.

Reply:
04d9d67d044